build.gradle

```diff
@@ -242,6 +242,7 @@ subprojects { subproject ->
     compileKotlin.dependsOn updateCopyrights
 
 	test {
+        jvmArgs '--add-opens java.base/java.util.zip=ALL-UNNAMED'
```

Comment:
Does it still work if I run tests from IDE?

Reply:
You have to add it to the run configuration for `BatchingRabbitTemplateTests`.

Reply:
Yeah... That's not I do on regular basis...
We might need to revise the logic to avoid reflection over there (if any).